kubernetes/overlays/authorization-management/configmap.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 data:
   POSTGRES_DB: "persistence_management"
   POSTGRES_SCHEMA: "{{NAMESPACE}}_authz"
-  CQRS_PROJECTION_COLLECTION_NAME: "clients"
+  CQRS_CLIENTS_PROJECTION_COLLECTION_NAME: "clients"
```

Comment:
not sure, it needs to be checked, but this rename may fail.
in `cqrs-projection.yaml` the `CQRS_PROJECTION_COLLECTION_NAME` env var is populated with the variable of the same name in the service configmap, but here it has been removed